### DISCUSSION TITLE
Any information on how to use the editor?

### DISCUSSION THREAD

**Discussion by ghost** [rokups/rbfx#280] [Q&A]
I've tried right clicking everything I could, and dragging/dropping anything onto everything else, and selecting things and pressing Enter/Spacebar, but I haven't yet figured out how to make the editor add something to the scene. How do I add models and materials and move/scale things? Or is the editor not designed to do these operations?

**Comment by eugeneko** [comment]
TL;DR: Editor is for brave.

I use this Editor quite often to edit scenes, but it's in very raw state. Hotkeys are broken, a lot of functionality missing, bugs, bugs, usability wanted.

I would say... treat the Editor as a tool to edit and build scenes. Model import works sometimes, other times it does not 🤷 

I will do something about Editor... at some point. For now, it's very WIP tool that you use at your own discretion.
I still find it useful despite being so raw - better than nothing.

Poke @rokups, he deserves it.

**Reply by ghost** [reply]
Okay thanks.

**Reply by ghost** [reply]
Now that I'm more familiar with the engine I'd like to try to build a scene with the editor. Any tips on how to get it into a mode where I can actually do something with a scene? I'm hoping for instructions on how to get the Editor into a state where I see a scene being rendered, and I can modify that scene.

**Reply by rokups** [reply]
Right-click in resources window and create new scene. Double-click created new scene to open it. Add nodes and components by right-clicking in hierarchy window or scene viewport.

**Reply by eugeneko** [reply]
@rokups +1 point to have actual default scene in editor.

**Reply by eugeneko** [reply]
@pcsagan FYI new editor is merged
